spec/features/merchants/discounts/user_discount_spec.rb

```diff
@@ -41,13 +41,37 @@
   end
 
   it "can get a discount on items" do
-    5.times do
+    10.times do
       click_link "#{@chain.name}"
       click_button "Add To Cart"
     end
-    click_link "Cart: 5"
-    expect(page).to have_content("#{@discount_1.name} applied!")
-    expect(page).to have_content("Total: $190.00")
-    # save_and_open_page
+    click_link "Cart: 10"
+    expect(page).to have_content("$380.00")
+    expect(page).to have_content("Total: $380.00")
```

Comment:
Test that the appropriate discount subtotal shows in the cart and the total is the same since there is no other items in the cart